frontend/packages/dev-console/src/components/import/deployImage-submit-utils.ts

```diff
@@ -246,8 +246,14 @@ export const createResources = (
     }
   } else if (!dryRun) {
     // Do not run serverless call during the dry run.
+    const [imageStreamResponse] = await Promise.all(requests);
     requests.push(
-      createKnativeService(name, projectName, formData.serverless.scaling, isiName, tag),
+      createKnativeService(
+        name,
+        projectName,
+        formData.serverless.scaling,
```

Comment:
Do you think you should deconstruct this so there there is less of a chance of merge conflicts?